server/api/plextv.ts

```diff
@@ -234,6 +234,9 @@ class PlexTvAPI extends ExternalAPI {
     }
   }
 
+  /**
+   * @deprecated Plex has deprecated '/pms/friends/all', use getUsers(); instead.
+   */
```

Comment:
We can just remove getFriends as it will no longer be used instead of adding this comment.